src/main/java/rx/internal/operators/OperatorOnBackpressureLatest.java

```diff
@@ -0,0 +1,225 @@
+
+import rx.Observable.Operator;
+import rx.*;
+
+/**
+ * An operator which drops all but the last received value in case the downstream
+ * doesn't request more.
+ */
+public final class OperatorOnBackpressureLatest<T> implements Operator<T, T> {
+    /** Holds a singleton instance initialized on class-loading. */
+    static final class Holder {
+        static final OperatorOnBackpressureLatest<Object> INSTANCE = new OperatorOnBackpressureLatest<Object>();
+    }
+    
+    /**
+     * Returns a singleton instance of the OnBackpressureLatest operator since it is stateless.
+     * @return the single instanceof OperatorOnBackpressureLatest
+     */
+    @SuppressWarnings("unchecked")
+    public static <T> OperatorOnBackpressureLatest<T> instance() {
+        return (OperatorOnBackpressureLatest<T>)Holder.INSTANCE;
+    }
+    
+    @Override
+    public Subscriber<? super T> call(Subscriber<? super T> child) {
+        final LatestEmitter<T> producer = new LatestEmitter<T>(child);
+        LatestSubscriber<T> parent = new LatestSubscriber<T>(producer);
+        producer.parent = parent;
+        child.add(parent);
+        child.add(producer);
+        child.setProducer(producer);
+        return parent;
+    }
+    /**
+     * A terminatable producer which emits the latest items on request.
+     * @param <T>
+     */
+    static final class LatestEmitter<T> extends AtomicLong implements Producer, Subscription, Observer<T> {
+        /** */
+        private static final long serialVersionUID = -1364393685005146274L;
+        final Subscriber<? super T> child;
+        LatestSubscriber<? super T> parent;
+        final AtomicReference<Object> value;
+        /** Written before done, read after done. */
+        Throwable terminal;
+        volatile boolean done;
+        /** Guarded by this. */
+        boolean emitting;
+        /** Guarded by this. */
+        boolean missed;
+        static final Object EMPTY = new Object();
+        static final long NOT_REQUESTED = Long.MIN_VALUE / 2;
+        public LatestEmitter(Subscriber<? super T> child) {
+            this.child = child;
+            this.value = new AtomicReference<Object>(EMPTY);
+            this.lazySet(NOT_REQUESTED); // not 
+        }
+        @Override
+        public void request(long n) {
+            if (n >= 0) {
+                for (;;) {
+                    long r = get();
+                    if (r == Long.MIN_VALUE) {
+                        return;
+                    }
+                    long u;
+                    if (r == NOT_REQUESTED) {
+                        u = n;
+                    } else {
+                        u = r + n;
+                        if (u < 0) {
+                            u = Long.MAX_VALUE;
+                        }
+                    }
+                    if (compareAndSet(r, u)) {
+                        if (r == NOT_REQUESTED) {
+                            parent.requestMore(Long.MAX_VALUE);
```

Comment:
So ... we are tracking the requested value for downstream, but upstream we just request MAX_VALUE the first time we receive a request, correct? Is this why BackpressureUtils doesn't work?